lib/inspec/cli.rb

```diff
@@ -393,6 +393,13 @@ def version
   end
   map %w{-v --version} => :version
 
+  desc "clear_cache", "clears the InSpec cache at ~/.inspec/cache. Useful for\
+  debugging."
+  def clear_cache
+    FileUtils.rm_r Dir.glob(File.expand_path("~/.inspec/cache/*"))
```

Comment:
This needs to work for Windows as well. 
I half remember the InSpec work dir (`~/.inspec`) being configurable as well.

Reply:
I see we do configure cache on `--vendor-cache` option so added support for that as well. If there's anything else I'm missing let me know as that's all I could see! 

From a product perspective if you like I could name it more generally to `--cache-path` instead to future proof this as well? 🤔 

I believe this passes on CI on windows, but on a cursory look it appears that FileUtils.rm_rf can be risky in relation to symlinking on windows machines? My windows-foo is not great, and I might need some guidance on detecting OS
from the `cli.rb` level- but would we be looking at something sort of like this?

```ruby
if somehow_determine_windows?
  system "rmdir /s/q #{o[:vendor_cache]}"
else
  FileUtils.rm_rf File.expand_path(o[:vendor_cache])
end
```